src/v8runtime/V8Runtime.cpp

```diff
@@ -576,6 +576,9 @@ jsi::PropNameID V8Runtime::createPropNameIDFromSymbol(
       static_cast<const V8PointerValue *>(getPointerValue(sym));
   assert(v8PointerValue->Get(isolate_)->IsSymbol());
```

Comment:
```suggestion
  assert(static_cast<const V8PointerValue *>(getPointerValue(sym))->Get(isolate_)->IsSymbol());
```

so that we could remove the `const V8PointerValue *v8PointerValue` entirely.